listenbrainz/webserver/views/stats_api.py

```diff
@@ -595,14 +596,15 @@ def get_artist_map(user_name: str):
                 raise APINoContent('')
         else:
             # Calculate the data
-            artist_msids = []
-            artist_mbids = []
+            artist_msids = defaultdict(lambda: 0)
+            artist_mbids = defaultdict(lambda: 0)
             top_artists = getattr(artist_stats, stats_range).artists
             for artist in top_artists:
                 if artist.artist_msid is not None:
-                    artist_msids.append(artist.artist_msid)
+                    artist_msids[artist.artist_msid] += artist.listen_count
```

Comment:
this would have been cleaner as `artist_msids[artist.artist_msid] = artist.listen_count`, or is there an edge case here that I'm missing?

Reply:
We may have multiple entries of the same MSIDs, we should count listens for all of those.